sips/sip-031/sip-031.md

```diff
@@ -0,0 +1,573 @@
+* Improved reliability and confidence in roadmap delivery via public commitments and long-term funding.  
+* Dedicated support for integrations (e.g., funding for dev time, audits, or infrastructure deployment).  
+* Expanded market activity from treasury-backed growth campaigns (e.g., new STX/sBTC listings, liquidity incentives).  
+* Lower organizational complexity when negotiating integration, marketing, or technical agreements.  
+* Active liquidity and incentive support to boost usage post-integration (especially valuable for exchanges and wallets).  
+* Stronger long-term alignment between infrastructure partners and ecosystem growth priorities.  
+* Greater brand lift as Stacks expands visibility and market cap, helping partners benefit from being early or core participants.
+
+### Casual Holders
+
+* Clear understanding of who does what in the ecosystem and where/how to get involved  
+* Increased token utility and use cases resulting in a stronger, more valuable ecosystem.  
+* Higher visibility of Stacks through global marketing and product announcements.  
+* Predictable emissions and inflation are aligned with long-term growth.  
+* Confidence that the ecosystem is addressing past inefficiencies and growing responsibly.  
+* Improved app and protocol quality through funded security, liquidity, and dev support.  
+* Trust in long-term alignment from professional treasury operations and public reporting.
+
+### **Immediate Action Post-SIP (if approved)**
+
+* The legal formation of both entities will be completed within 60 days of SIP approval. The research for this proposal has resulted in a significant head start on these requirements.  
+* The Appointments Committee will fill the remaining seats based on community nominations, opening these nominations within 60 days of SIP approval.  
+* Leadership searches for the Stacks Endowment (CIO) and operational entity (CEO) will commence, with positions posted publicly within 60 days.  
+* The new operational entity, in collaboration with the Stacks Endowment and Stacks Builders will finalize details of a retroactive grants program that will focus on growth, enabling predictable capital relative to measurable impact to the ecosystem, such as user, TVL, or other growth indicators.  
+* Grant applications for critical work in the very short term will be opened by the Stacks Endowment, likely leveraging support from the Stacks Foundation’s team and systems. The goal of these grants is to serve any short-term critical needs or opportunities in the initial 90 days of the Endowment's existence until a more formal program can be deployed, including the aforementioned retroactive grant program.
+
+## **Voting Timeline**
+
+The voting timeline will be finalized via community discussion. Pending final results from the Appointments Committee, current sentiment is that voting could start June 24. 
+
```

Comment:
I think this can be removed. once the timeline is chosen, it should be specified in the Activation section

Reply:
Sounds good

Reply:
```suggestion

```